service/co/hotwax/alc/ArtifactLifeCycleServices.xml

```diff
@@ -34,7 +34,7 @@
             <!-- Iterate through the list of artifact logs and process each one -->
             <set field="createIndexCount" value="0" type="Integer"/>
             <iterate list="artifactLogList" entry="artifactEvent">
-                <service-call name="co.hotwax.alc.ArtifactLifeCycleServices.create#ArtifactLogIndex" in-map="[artifactEventId:artifactEvent.artifactEventId]"/>
+                <service-call name="co.hotwax.alc.ArtifactLifeCycleServices.create#ArtifactLogIndex" in-map="[artifactEventId:artifactEvent.artifactEventId]" ignore-error="true" transaction="force-new"/>
```

Comment:
- `ignore-error="true"`: It is a temporary fix to continuing the loop in case of any error in the current iteration
- `transaction="force-new"`: It prevents the whole transaction rollback, in case there is an error occurs in the core service